example/java/build.gradle

```diff
@@ -0,0 +1,61 @@
+apply plugin: 'java'
+apply plugin: 'com.google.protobuf'
```

Comment:
should be after buildscript

Reply:
Why not put all all "apply" in one place?

Reply:
Hm, I thought it doesn't work, sorry